lib/redshift_ecto.ex

```diff
@@ -95,7 +95,7 @@ defmodule RedshiftEcto do
   """
 
   # Inherit all behaviour from Ecto.Adapters.SQL
-  use Ecto.Adapters.SQL, :postgrex
+  use Ecto.Adapters.SQL, driver: :postgrex, migration_lock: nil
```

Comment:
migration_lock is intended to create a read lock on the schema migrations table so that migrations can be run concurrently, but only one runner proceeds. For postgres and mysql, it uses `FOR UPDATE` as in `SELECT * FROM schema_migrations FOR UPDATE`. I didn't check if redshift has something equivalent.